x-pack/plugins/triggers_actions_ui/README.md

```diff
@@ -206,13 +206,13 @@ export const IndexThresholdAlertTypeExpression: React.FunctionComponent<IndexThr
   ....
 
```

Comment:
Should we add guidance in the README about using React components for logos instead of directly importing SVGs?

Reply:
I will add this